src/taco-cli/cli/utils/cordovaHelper.ts

```diff
@@ -315,6 +318,23 @@ class CordovaHelper {
             });
     }
 
+    public static getCordovaExecutable(): Q.Promise<string> {
+        return projectHelper.getProjectInfo()
+            .then(function (projectInfo: projectHelper.IProjectInfo): any {
+                if (projectInfo.cordovaCliVersion) {
+                    return packageLoader.lazyRun(CordovaHelper.CORDOVA_NPM_PACKAGE_NAME, CordovaHelper.CORDOVA_NPM_PACKAGE_NAME + "@" + projectInfo.cordovaCliVersion, "cordova")
+                        .then(function (cordovaExecutablePath: string) {
+                            return cordovaExecutablePath;
+                        });
+                } else {
+                    return CordovaHelper.globalCordovaCommandName;
```

Comment:
Not sure if I should reject here so we have a single place to handle returning the global cordova command.